unitTests/ui/Combobox/ComboboxValuesTest.ts

```diff
@@ -320,6 +320,108 @@ export function ComboboxValuesTest() {
       });
     });
 
+    describe('when calling "focusFirstValue"', () => {
+      beforeEach(() => {
+        triggerRenderFromResponse(['hi', 'hello', 'bye']);
+      });
+
+      it(`when no value is active
+      should activate the first value`, () => {
+        comboboxValues.focusFirstValue();
+
+        expect(isChildrenActiveAtIndex(0)).toBe(true);
+      });
+
+      it(`when no value is active
+      should focus the first value`, () => {
+        comboboxValues.focusFirstValue();
+
+        expect(getFocusCountAtIndex(0)).toEqual(1);
+      });
+
+      it(`when a value is active
+      should activate the first value`, () => {
+        comboboxValues.focusNextValue();
+        comboboxValues.focusNextValue();
+        comboboxValues.focusFirstValue();
+
+        expect(isChildrenActiveAtIndex(1)).toBe(false);
+        expect(isChildrenActiveAtIndex(0)).toBe(true);
+      });
+
+      it(`when a value is active
+      should focus the first value`, () => {
+        comboboxValues.focusNextValue();
+        comboboxValues.focusNextValue();
+        comboboxValues.focusFirstValue();
+
+        expect(getFocusCountAtIndex(0)).toEqual(2);
+        expect(getFocusCountAtIndex(1)).toEqual(1);
+      });
+
+      it('should call "updateAccessibilityAttributes" with the rigth attributes', () => {
```

Comment:
Small typo in the test description here: `rigth` -> `right`.
Same for line 412.